.github/workflows/test.yml

```diff
@@ -82,7 +82,8 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: '1.20'
+          args: --timeout=10m
```

Comment:
Isn't 10m quite long ?

Reply:
just try to see if it works 🤔

Reply:
unfortunately it doesn't work

Reply:
Ah I added it to a wrong place. Let me try it again!